tests/wordcount.js

```diff
@@ -158,6 +158,23 @@ describe( 'WordCount', () => {
 					.then( done )
 					.catch( done );
 			} );
+
+			describe( 'container not created', () => {
+				let editor;
+
+				beforeEach( () => {
+					return VirtualTestEditor.create( {
+						plugins: [ WordCount, Paragraph ]
+					} )
+						.then( _editor => {
+							editor = _editor;
+						} );
+				} );
+
+				it( 'should not throw an error', () => {
+					return editor.destroy();
+				} );
+			} );
```

Comment:
This test scenario could be simplified:

```js
it( 'does not throw any error if the container is not specified', () => {
	return VirtualTestEditor
		.create( {
			plugins: [ WordCount, Paragraph ]
		} )
		.then( editor => {
			return editor.destroy();
		} );
} );
```

The whole `describe()` block could be replaced with the snippet.

Reply:
I was thinking about it too. If we have only one test we can simplify it.

Reply:
I just copied logic from above. I simplified the logic as @pomek suggest.